### ISSUE TITLE
Feature request: light theme

### ISSUE THREAD

**Issue by ralaud** [hardcore-sushi/DroidFS#84]
I am using light theme on my phone. Due to I would like to use light theme in DroidFS too. It would be just nice to have. But importance is not so high.

Best regards!

**Comment by hardcore-sushi** [comment]
Yeah but I try to be a little green, I don't want to contribute to energy wasting caused by white pixels. I won't implement the feature myself, but I would accept pull requests. After all, that's an open-source project.

**Comment by dejafait** [comment]
+1 on adding light theme as an option instead of forcing dark theme on everyone. Some individuals might have a good reason to prefer light theme. I personally have a disability and I cannot use backlit screens anymore, I use eink screens all the time. On eink screens light theme looks excellent, just like physical books, however dark theme looks terrible. I respect your wish to reduce energy consumption and I believe you are doing your part by setting default to dark. However setting default and forcing dark are two different things. I love the DroidFS app that I use daily, thank you very much for a great piece of software 😊

**Comment by hardcore-sushi** [comment]
v2.3.0 introduces a "Dynamic" color theme that adapts to the device's current dark theme preference. If you also want light variants of classic themes, feel free to open a PR.
